src/chips/nrf51822/chip.rs

```diff
@@ -0,0 +1,42 @@
+use common::{RingBuffer,Queue};
```

Comment:
You're not using interrupts, and this file is only set up to include interrupts from the nrfblinky branch. You shouldn't include this.